test/e2e_flags.go

```diff
@@ -50,8 +52,12 @@ func initializeFlags() *EnvironmentFlags {
 	flag.StringVar(&f.DockerRepo, "dockerrepo", defaultRepo,
 		"Provide the uri of the docker repo you have uploaded the test image to using `uploadtestimage.sh`. Defaults to $DOCKER_REPO_OVERRIDE")
 
-	usr, _ := user.Current()
-	defaultKubeconfig := path.Join(usr.HomeDir, ".kube/config")
+	defaultKubeconfig := "kubeconfig"
+	if usr, err := user.Current(); err != nil {
+		log.Printf("Error getting current user, using %s as fallback: %v", defaultKubeconfig,  err)
```

Comment:
For consistency, I think it would be better using the context logger instead of the vanilla `log`.

Reply:
Logging.go uses glog to log before contextual logger is initialized.